test/plugin/test_in_kubernetes_metrics_aggregator.rb

```diff
@@ -43,7 +43,7 @@ class KubernetesMetricsAggInputTest < Test::Unit::TestCase
     ENV['KUBERNETES_SERVICE_PORT'] = "10255"
     stub_k8s_requests
     @driver = create_driver(METRIC_TEST_CONFIG)
-    @driver.run timeout: 20, expect_emits: 200, shutdown: false
+    @driver.run timeout: 30, expect_emits: 200, shutdown: false
```

Comment:
Does the timeout need to be changed?
Can you provide us info as to why we are doing this?

Reply:
I don't know how the test framework actually works and why tests are so slow.
I have assumed there is some sort of default interval at which the metrics are collected that did not trigger again once I added the exception handling. Thus I've added some extra time and they did pass. It might not be deterministic, hard for me to tell. 

It is hard for me to analyse, because the installation of the project dependencies failed on my machine. Hence I was not able to run the code locally and tested the code by connecting CircleCI to my fork.

Reply:
Thats cool, did you deploy it in a pod and test it in kubernetes?

Reply:
No, I did not check that in Kubernetes